static/lang/action-en.json

```diff
@@ -692,6 +692,7 @@
                     "RiskySurgery": "Risky Surgery"
                 },
                 "Label": "Treat Wounds",
+                "MedicineDC": "Medicine DC:",
```

Comment:
use "PF2E.InlineCheck.DCWithName"

Reply:
If we do, we won't get the colon on the end.

PF2E.Actions.TreatWounds.MedicineDC:
<img width="409" alt="Screen Shot 2023-06-20 at 4 22 08 PM" src="https://github.com/foundryvtt/pf2e/assets/1904698/e851e4c0-48e9-4547-9442-b6611b5c3bf1">

PF2E.InlineCheck.DCWithName:
<img width="409" alt="Screen Shot 2023-06-20 at 4 24 47 PM" src="https://github.com/foundryvtt/pf2e/assets/1904698/b779294f-ea00-4280-8271-201244f905a1">

Reply:
Then remove the others.